radio/src/gui/gui_common.h

```diff
@@ -184,6 +184,28 @@ inline uint8_t MODULE_CHANNELS_ROWS(int moduleIdx)
   }
 }
 
+inline uint8_t IF_ALLOW_RACING_MODE(int moduleIdx, uint8_t flag)
+{
+  static uint8_t unenableCount = 0;
+  if (flag && moduleIdx==INTERNAL_MODULE) {
+    if (!IS_MODULE_ENABLED(moduleIdx)) {
+      return HIDDEN_ROW;
+    }
+    else if (isModulePXX2(moduleIdx)) {
+      if (0 == g_model.moduleData[moduleIdx].channelsCount)
+        return 1;
+    }
+    g_model.moduleData[moduleIdx].pxx2.enableRacingMode = 0;
```

Comment:
This changes something in your model, depending if you are inside the menu or not!